content/interface.md

```diff
@@ -1,30 +1,40 @@
-# The JupyterLab and notebook interface
-
-```{questions}
-- How does the JupyterLab interface work?
-- How do I edit Jupyter notebooks?
-- How can I version control notebooks?
-```
+# JupyterLab and notebook interface
 
 ```{objectives}
 - Learn to navigate JupyterLab user interface.
 - Discuss integrated development environments.
 - Get an overview of useful keyboard shortcuts.
-- Learn about command/edit modes, markdown/code cells and Git integration.
+- Learn about command/edit modes and markdown/code cells.
 ```
 
 JupyterLab is the next-generation user interface for Jupyter notebooks
 and is intended to replace the conventional interface.
 It is a highly modular and customizable interface.
 
+---
+
+## How to start JupyterLab
+
 Let's have a look at how it works. We go to terminal, and type:
-```shell
+```console
 $ mkdir jupyterlab-demo
 $ cd jupyterlab-demo
 $ jupyter-lab
 ```
 - On Windows, the JupyterLab App can also be launched by clicking on the JupyterLab icon in the Anaconda menu.
```

Comment:
according they have installed it. I don't think I have it with miniconda3.

Reply:
Ah good catch.